node/src/main/scala/coop/rchain/node/configuration/model.scala

```diff
@@ -100,6 +102,10 @@ final case class Metrics(
     sigar: Boolean
 )
 
+final case class DevConf(
+    deployerPrivateKey: Option[String]
+)
```

Comment:
Maybe _enabled_ field can be part of `DevConf` instead of _devMode_ option on root config?

Reply:
I'm not sure actually, @zsluedem wdy think?

Reply:
Oh. I actually did some comments a few days before. But it is gone. 
Normally I agree with @tgrospic .